<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditStatusDialogClass</class>
 <widget class="QDialog" name="EditStatusDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit statuses</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QTableWidget" name="tblStatus">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbtAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dbtDialogButtons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tblStatus</tabstop>
  <tabstop>pbtAdd</tabstop>
  <tabstop>pbtDelete</tabstop>
  <tabstop>dbtDialogButtons</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dbtDialogButtons</sender>
   <signal>rejected()</signal>
   <receiver>EditStatusDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
